mmv1/third_party/terraform/services/resourcemanager/data_source_google_active_folder.go

```diff
@@ -26,6 +27,13 @@ func DataSourceGoogleActiveFolder() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"rest_method": {
```

Comment:
non-blocking: what would you think about calling this `api_method` rather than `rest_method`? Although it's called a REST API, SEARCH is not a standard REST method...